app/scripts/controllers/network/network-controller.ts

```diff
@@ -108,44 +108,44 @@ export enum NetworkControllerEventType {
  * switched, but the new provider has not been created and no state changes have
  * occurred yet.
  */
-type NetworkControllerNetworkWillChangeEvent = {
+export type NetworkControllerNetworkWillChangeEvent = {
   type: NetworkControllerEventType.NetworkWillChange;
-  payload: [];
+  payload: never[];
```

Comment:
Why would `never` be used here? I'd expect `never` to only ever show up in conditional, unreachable branches. Having it as a property implies that this type is never used, doesn't it?